pkg/apis/serving/k8s_validation.go

```diff
@@ -267,7 +284,58 @@ func ValidatePodSpec(ps corev1.PodSpec) *apis.FieldError {
 	return errs
 }
 
+// ValidateMultiContainerPorts validates port when specified multiple containers
```

Comment:
any reason for this to be a  public function?

Reply:
As of now its used only in this function but later when i add logic for multi container These functions will be used in another package.

But in this PR its not used so i will make it to small